src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -863,3 +863,30 @@ def get(self, vol_name: str, subvol_name, group_name: str = '', info=True):
                     )
                 snapshot['info'] = json.loads(out)
         return snapshots
+
+@APIRouter('/cephfs/snaphost/schedule', Scope.CEPHFS)
+@APIDoc("Cephfs Snapshot Scheduling API", "CephFSSnapshotSchedule")
+class CephFSSnapshotSchedule(RESTController):
+    
+    def list(self, path: str = '/', recursive: bool = True, fs: str = None):
+        error_code, out, err = mgr.remote('snap_schedule', 'snap_schedule_list', path, recursive, fs, 'plain')
```

Comment:
shouldn't this be json? because i see a lot of json processing going on here. at the end an `output_json` is generated by dumping the json and eventually that's being loaded into the parser to return the json.  output
```suggestion
        error_code, out, err = mgr.remote('snap_schedule', 'snap_schedule_list', path, recursive, fs, 'json')
```

Reply:
@nizamial09 
the json output doesn't return the list of paths and I need it to get all the snap schedule details from ```snap_schedule_get```.